habitat_sim/utils/data/data_extractor.py

```diff
@@ -31,15 +36,28 @@ class ImageExtractor:
 
     def __init__(
         self,
-        scene_filepath,
+        filepath,
```

Comment:
maybe `scenes_filepath`, `scene` context is important.

Reply:
Fixing this up now